libs/gpu-dawn/sdk.zig

```diff
@@ -43,6 +43,9 @@ pub fn Sdk(deps: anytype) type {
             /// Whether to build Dawn from source or not.
             from_source: bool = false,
 
+            /// Produce static libraries at zig-out/lib
+            install_libs: bool = false,
```

Comment:
We need to update `libs/gpu-dawn/build.zig` to pass `true` for this option.

Otherwise when the `gpu-dawn` repository CI runs, it won't produce `zig-out/lib/libdawn.a` and the scripts that upload the gpu-dawn binaries will fail.

Reply:
hmm this doesn't make sense. maybe it's better to add an option like `dawn-from-source` and update script?